release_notes.md

```diff
@@ -1,5 +1,5 @@
 ## New Features
```

Comment:
Can we remove the "Bug Fixes" and "Breaking Changes" sections since they don't have any items?

Reply:
As an alternative, I will add comments saying that we don't have any items for these categories.

Reply:
Actually, for "Bug Fixes", I'll list all the bug fixes that are coming in as part of the DurableTask.AzureStorage dependency updates here. For breaking changes, I'll just make a note to "See the dependency updates below" since those can *technically* be considered breaking.

Reply:
Ok, that would be great!